integration-test/src/main/java/com/sequenceiq/it/IntegrationTestApp.java

```diff
@@ -168,6 +171,7 @@ private XmlSuite loadSuite(String suitePath, InputStreamSource resource) throws
             XmlSuite xmlSuite = parser.parse(suitePath, inputStream, true);
             xmlSuite.setParallel(XmlSuite.ParallelMode.getValidParallel(parallel));
             xmlSuite.setThreadCount(threadCount);
+            xmlSuite.setTimeOut(timeOut);
```

Comment:
Is this a suite timeout or individual testcase timeout?

Reply:
Hi, Based on my knowledge the timeout for tests. You can define this at suite or test level. However this always means test timeout. If you define a different timeout for a specified test, you can override the test suite setting also.